src/runtime.js

```diff
@@ -22,6 +27,15 @@ let tornado = {
   },
 
   /**
+   * Register a helper, overwrite if a helper already exists
+   * @param {String} name The name of the helper
+   * @param {Function} method The function to be executed when the helper is found in a template
+   */
+  registerHelper(name, method) {
```

Comment:
Also, it'd be nice to pass in an object so we don't have to individually register helpers.  Maybe another function:

let myCompanyHelpers = {
  nameFormat: function(x, x, x),
  dateFormat: function(x,x,x)
};

td.registerHelpers(myCompaniesHelpers);